src/main/kotlin/at/bitfire/dav4jvm/property/push/AuthSecret.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package at.bitfire.dav4jvm.property.push
+
+import at.bitfire.dav4jvm.Property
+import at.bitfire.dav4jvm.PropertyFactory
+import at.bitfire.dav4jvm.XmlReader
+import org.xmlpull.v1.XmlPullParser
+
+/**
+ * Represents a `{DAV:Push}auth-secret` property.
```

Comment:
Can we still use `DAV:Push` in `{DAV:Push}auth-secret` here? I am wondering since the namespace changed to `https://github.com/bitfireAT/webdav-push/`.

Reply:
We'd need to change that in the other remaining kdoc too.

Reply:
> Can we still use DAV:Push in {DAV:Push}auth-secret here?

I thought about that, but the whole URL is a bit too much for me on the comment, no?

Reply:
Well, yes it's long, but like this right it's simply incorrect, or not? I would always prefer a correct comment over a short and wrong comment.

Reply:
Maybe

> WebDAV-Push `auth-secret`

?

Reply:
I've finally used `[NS_WEBDAV_PUSH]:web-push-subscription`, it renders quite nicely in the IDE, and allows direct access to it:

![image](https://github.com/user-attachments/assets/2ef63c07-d421-4fb4-93b6-2fcc89df4f5c)